docs/source/pre_application/testnet.rst

```diff
@@ -13,3 +13,25 @@ You can view it on `etherscan <https://rinkeby.etherscan.io/address/0xc3871e2c11
 .. attention::
 
     This testnet is currently being migrated to Threshold Network.
+
+
+Stake and Bond
+--------------
+You need some rinkeby testnet ETH and 40,000 T in a metamask wallet.
+Ask in the discord if you need help with this.
+
+Navigate to the `Testnet Staking Dashboard <https://dn3gsazzaajb.cloudfront.net/manage/stake>`_ and connect your metamask wallet.
```

Comment:
this should not be public

Reply:
It's part of the ibex staking guide, I've just relocated it.  We can completely remove it when we've validated and documented the Oryx testnet flow, hopefully by the time this PR is RFR.

Reply:
✔️